### PR TITLE
add ui-sorter

### PR DESCRIPTION
**Pull request by null-null-null**
@pzuraq 

Support all the way back to IE8! But regrettably, no integration tests. If you know of a good way to simulate drag and drop events, let me know.

### CODE REVIEW

File: addon/components/ui-sorter-item.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+import layout from '../templates/components/ui-sorter-item';
+
+const { Component, on } = Ember;
```
Comment (pzuraq) [inline addon/components/ui-sorter-item.js]: on isn't used

**Comment by pzuraq** [comment]
Looks good so far, the styles could use a little work. Ideally we'd want the picked element to disappear and to have the remaining elements move to make space for the dropzone. I get the feeling this can be added with CSS classes.

I'll take a look at the CSS and see if I can add this as default styling with our tf classes.

**Comment by pzuraq** [comment]
Looks like it's doable, may require some work though. We should reverse engineer this https://github.com/voidberg/html5sortable

**Comment by pzuraq** [comment]
@null-null-null Did some work to get that better UX, take a look and merge when ready.

**Comment by null-null-null** [comment]
@pzuraq Looks great on Chrome, but FF is buggy. Added a line that should help with part of the FF problem, but it didn't fix everything. I'll look into this more later today

**Pull request merged**